ubuntu/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM ubuntu
+FROM ubuntu:18.10
```

Comment:
I'd prefer to keep this generic, but if we move to use a particular version, an LTS release (18.04 in this case) would be preferred.

Reply:
One of the aspects I found confusing when working on the development was that the same dockerfile was used for the ubuntu16 image, but I didn't even know what specific version. I would argue that putting an explicit version improves the reproducability, but at the end of the day, this is your package, so if you prefer to put a generic version, that's fine with me.

Reply:
I understand. The initial idea of these Dockerfiles were to serve as templates for others to use in their ROOT-based images. However, I see that more and more people use these in CI setups, so I am going to work on automating more images, tagging by ROOT version, etc, and it probably also makes sense to move to specific versions for the OS. In the case of Ubuntu, I think sticking with the latest LTS would be best.  I actually have some local files where I am adapting for the different versions of ubuntu, so we could have, e.g. `Dockerfile.{xenial,bionic,cosmic,disco,...}` or a generic Dockerfile that takes the version as parameter and has the latest LTS as default.